Native issue with JavaParser

```
Caused by: com.sun.tools.javac.util.FatalError: Fatal Error: Unable to find package java.lang in classpath or bootclasspath
	at com.sun.tools.javac.comp.TypeEnter$ImportsPhase.resolveImports(TypeEnter.java:354) ~[na:na]
	at com.sun.tools.javac.comp.TypeEnter$ImportsPhase.runPhase(TypeEnter.java:321) ~[na:na]
	at com.sun.tools.javac.comp.TypeEnter$Phase.doCompleteEnvs(TypeEnter.java:282) ~[na:na]
	at com.sun.tools.javac.comp.TypeEnter$Phase.completeEnvs(TypeEnter.java:251) ~[na:na]
	at com.sun.tools.javac.comp.TypeEnter.complete(TypeEnter.java:198) ~[na:na]
	at com.sun.tools.javac.code.Symbol.complete(Symbol.java:642) ~[na:na]
	at com.sun.tools.javac.code.Symbol$ClassSymbol.complete(Symbol.java:1326) ~[na:na]
	at com.sun.tools.javac.comp.Enter.complete(Enter.java:583) ~[na:na]
	at com.sun.tools.javac.comp.Enter.main(Enter.java:560) ~[na:na]
	at org.openrewrite.java.Java11Parser.enterAll(Java11Parser.java:280) ~[na:na]


error listing module locations for SYSTEM_MODULES: java.io.IOException: can't find system classes
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -40,6 +40,9 @@ jobs:
           - nickname: linux
             os: ubuntu-latest
             graal: 22.0.0.2
+          - nickname: linux-dev
+            os: ubuntu-latest
+            graal: 22.0.0.2
     name: CI Native ${{ matrix.graal }} on ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v2

--- a/src/main/java/org/springframework/up/command/BootCommands.java
+++ b/src/main/java/org/springframework/up/command/BootCommands.java
@@ -24,6 +24,7 @@ import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
+import java.util.function.Consumer;
 
 import org.apache.maven.model.Model;
 import org.apache.tika.Tika;
@@ -32,6 +33,7 @@ import org.codehaus.plexus.util.DirectoryScanner;
 import org.jetbrains.annotations.Nullable;
 import org.jline.utils.AttributedStringBuilder;
 import org.jline.utils.AttributedStyle;
+import org.openrewrite.InMemoryExecutionContext;
 import org.openrewrite.Recipe;
 import org.openrewrite.SourceFile;
 import org.openrewrite.java.ChangePackage;
@@ -296,7 +298,11 @@ public class BootCommands {
 		catch (IOException e) {
 			throw new UpException("Failed reading files in " + workingPath, e);
 		}
-		List<? extends SourceFile> compilationUnits = javaParser.parse(collector.getMatches(), null, null);
+		Consumer<Throwable> onError = e -> {
+			logger.error("error in javaParser execution", e);
+		};
+		InMemoryExecutionContext executionContext = new InMemoryExecutionContext(onError);
+		List<? extends SourceFile> compilationUnits = javaParser.parse(collector.getMatches(), null, executionContext);
 		ResultsExecutor container = new ResultsExecutor();
 
 		Recipe recipe = new ChangePackage(fromPackage, targetPackageName, true);

--- a/src/main/java/org/springframework/up/config/UpCliNativeConfiguration.java
+++ b/src/main/java/org/springframework/up/config/UpCliNativeConfiguration.java
@@ -24,10 +24,15 @@ import com.sun.jna.Pointer;
 import com.sun.jna.Structure;
 import com.sun.jna.ptr.IntByReference;
 import com.sun.jna.ptr.PointerByReference;
+import org.kohsuke.github.GHCommit;
+import org.kohsuke.github.GHLicense;
 import org.kohsuke.github.GHMyself;
 import org.kohsuke.github.GHObject;
 import org.kohsuke.github.GHPerson;
+import org.kohsuke.github.GHRepository;
 import org.kohsuke.github.GHUser;
+import org.kohsuke.github.GHVerification;
+import org.kohsuke.github.GitUser;
 
 import org.springframework.nativex.hint.FieldHint;
 import org.springframework.nativex.hint.JdkProxyHint;
@@ -68,13 +73,21 @@ import org.springframework.up.initializr.model.ProjectType.ProjectTypeValue;
  * @author Janne Valkealahti
  */
 @NativeHint(
-	resources = @ResourceHint(
-		patterns = {
-			"completion/.*",
-			"template/.*.st",
-			"org/springframework/shell/component/.*.stg",
-			"com/sun/jna/win32-x86-64/jnidispatch.dll"
-		}),
+	options = {
+		// https://github.com/rd-1-2022/spring-up/issues/14
+		"-H:+AllowJRTFileSystem"
+	},
+	resources = {
+		@ResourceHint(
+			patterns = {
+				"completion/.*",
+				"template/.*.st",
+				"org/springframework/shell/component/.*.stg",
+				"com/sun/jna/win32-x86-64/jnidispatch.dll",
+				"org/apache/tika/mime/tika-mimetypes.xml"
+			}
+		),
+	},
 	types = {
 		@TypeHint(
 			types = {
@@ -228,7 +241,17 @@ import org.springframework.up.initializr.model.ProjectType.ProjectTypeValue;
 			}
 		),
 		@TypeHint(
-			types = { GHMyself.class, GHObject.class, GHPerson.class, GHUser.class },
+			typeNames = "org.kohsuke.github.GitHubInteractiveObject",
+			access = {
+				TypeAccess.PUBLIC_CONSTRUCTORS, TypeAccess.DECLARED_CLASSES, TypeAccess.DECLARED_CONSTRUCTORS,
+				TypeAccess.DECLARED_FIELDS, TypeAccess.DECLARED_METHODS
+			}
+		),
+		@TypeHint(
+			types = {
+				GHMyself.class, GHObject.class, GHPerson.class, GHUser.class, GHCommit.class, GHLicense.class,
+				GHRepository.class, GHVerification.class, GitUser.class
+			},
 			access = {
 				TypeAccess.PUBLIC_CONSTRUCTORS, TypeAccess.DECLARED_CLASSES, TypeAccess.DECLARED_CONSTRUCTORS,
 				TypeAccess.DECLARED_FIELDS, TypeAccess.DECLARED_METHODS

--- a/src/main/resources/META-INF/native-image/resource-config.json
+++ b/src/main/resources/META-INF/native-image/resource-config.json
@@ -1,0 +1,16 @@
+{
+  "bundles":[
+    {
+      "name":"com.sun.tools.javac.resources.compiler",
+      "classNames":["com.sun.tools.javac.resources.compiler"]
+    },
+    {
+      "name":"com.sun.tools.javac.resources.ct",
+      "classNames":["com.sun.tools.javac.resources.ct"]
+    },
+    {
+      "name":"com.sun.tools.javac.resources.javac",
+      "classNames":["com.sun.tools.javac.resources.javac"]
+    }
+  ]
+}

